### DISCUSSION TITLE
硬链接方式转移成功，检查媒体库文件发现是copy而不是ln

### DISCUSSION THREAD

**Discussion by xmwxj** [jxxghp/MoviePilot#1676] [问题咨询] [answered]
使用硬链接的方式整理文件后，日志没有报错，历史记录提示成功转移。但是使用ls -l查看文件时，发现文件的引用是1，而不是2，说明是以copy的方式进行转移，而不是ln(链接)的方式进行转移。
![1](https://github.com/jxxghp/MoviePilot/assets/31843185/c83e4730-831f-4abb-a290-d3501111476e)
![2](https://github.com/jxxghp/MoviePilot/assets/31843185/ef83071f-f2bf-4b3d-b90f-2c9718ba99e5)
![3](https://github.com/jxxghp/MoviePilot/assets/31843185/60ddc7d6-7a91-4a94-9372-41ef291c79c7)

**Comment by cddjr** [comment, marked as answer]
可能是因为路径跨盘导致

**Reply by xmwxj** [reply]
确实是，在容器里面下载目录和媒体库目录分别映射成/download和/media，这个容器里面估计是跨盘了。我把根目录/disk1 映射到容器，下载目录和媒体库设置成/disk1/download,/disk1/media 就解决了。谢谢。
